lib/tapioca/generator.rb

```diff
@@ -253,13 +265,38 @@ def load_dsl_generators
 
     sig { params(constant_names: T::Array[String]).returns(T::Array[Module]) }
     def constantize(constant_names)
-      constant_names.map do |name|
+      constant_map = constant_names.map do |name|
         begin
-          name.constantize
+          [name, name.constantize]
         rescue NameError
-          nil
+          [name, nil]
+        end
+      end.to_h
+
+      unprocessable_constants = constant_map.select { |_, v| v.nil? }
+      unless unprocessable_constants.empty?
+        unprocessable_constants.each do |name, _|
+          say("Error: Cannot find constant '#{name}'", :red)
```

Comment:
Question: in which scenario would the file exist but the constant be missing? Is this in the case of renaming a class and forgetting to remove/change the corresponding RBI?

Reply:
I believe this one is about the constant you pass to Tapioca: `tapioca dsl MyConstant`

Reply:
Yes, if a constant that previously existed was removed or renamed, then the RBI could have been left dangling. For the full run, that will be cleaned up at the end of the run. I also wanted to make sure that the RBI is cleaned up even if someone supplied the old constant name explicitly.

Reply:
Btw, most people don't remove RBI files of constants that are removed or renamed, so this does happen a lot in our codebase.